app/assets/stylesheets/application/homepage/_event_thumbnail.scss

```diff
@@ -0,0 +1,55 @@
+.event-thumbnail {
+  background-color: $secondary-background-color;
+  vertical-align: top;
```

Comment:
Properties should be ordered background-color, display, margin, position, text-align, vertical-align, width